src/Apps/Order/Components/CreditCardInput.tsx

```diff
@@ -15,6 +15,8 @@ export const StyledCardElement = styled(CardElement)`
 // Re-uses old input border behavior
 const StyledBorderBox = styled(BorderBox).attrs<InputBorderProps>({})`
   ${inputBorder};
+  padding: 9px 10px;
```

Comment:
I adjusted this down from `padding: 10px` set by `inputBorder` so that this input's height is consistent with our other inputs on `/shipping` - we're using `40px` height right now it looks like.